website/docs/reference/resource-configs/firebolt-configs.md

```diff
@@ -0,0 +1,299 @@
+    ]
+) }}
+```
+
+### How Aggregating Indexes and Join Indexes Are Named
+
+In dbt-firebolt, you do not provide names for aggregating indexes and join indexes; they are named programmatically. dbt will generate index names using the following convention:
+```
+<table-name>__<key-column>__<index-type>_<unix-timestamp-at-execution>
+```
+
+For example, a join index could be named `my_users__id__join_1633504263` and an aggregating index could be named `my_orders__order_date__aggregating_1633504263`.
+
+### Managing Ingestion via External Tables
+
+`dbt-firebolt` supports dbt's [external tables feature](https://docs.getdbt.com/reference/resource-properties/external), which allows dbt to manage the table ingestion process from S3 into Firebolt. This is an optional feature but can be highly convenient depending on your use case.
+
+More information on using external tables including properly configuring IAM can be found in the Firebolt [documentation](https://docs.firebolt.io/sql-reference/commands/ddl-commands#create-external-table).
+
+#### Installation of External Tables Package
+
+To install and use `dbt-external-tables` with Firebolt, you must:
+
+1. Add this package to your packages.yml:
+
+    ```yaml
+    packages:
+      - package: dbt-labs/dbt_external_tables
+        version: <version>
+    ```
+
+2. Add these fields to your `dbt_project.yml`:
+
+    ```yaml
+    dispatch:
+      - macro_namespace: dbt_external_tables
+        search_order: ['dbt', 'dbt_external_tables']
+    ```
+
+3. Pull in the `packages.yml` dependencies by calling `dbt deps`.
+
+#### Using External Tables
+
+To use external tables, you must define a table as `external` in your `dbt_project.yml` file. Every external table must contain the fields `url`, `type`, and `object_pattern`. Note that the Firebolt external table specification requires fewer fields than what is specified in the dbt documentation.
+
+In addition to specifying the columns, an external table may specify partitions. Partitions are not columns and they cannot have the same name as columns. To avoid yaml parsing errors, remember to encase string literals (such as the `url` and `object_pattern` values) in single quotation marks.
+
+#### dbt_project.yml Syntax For an External Table
+
+```yaml
+sources:
+  - name: firebolt_external
+    schema: "{{ target.schema }}"
+    loader: S3
+
+    tables:
+      - name: <table-name>
+        external:
+          url: 's3://<bucket_name>/'
+          object_pattern: '<regex>'
+          type: '<type>'
+          credentials:
+            internal_role_arn: arn:aws:iam::id:<role>/<bucket-name>
+            external_role_id: <external-id>
+          object_pattern: '<regex>'
+          compression: '<compression-type>'
+          partitions:
+            - name: <partition-name>
+              data_type: <partition-type>
+              regex: '<partition-definition-regex>'
+          columns:
+            - name: <column-name>
+              data_type: <type>
+```
```

Comment:
we've already gutted most of the "using dbt-external tables" section, I vote we get rid of the rest of it?
```suggestion
```